content/docs/getting-started/installing.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Installing
+weight: 2
+---
+
+## From extension stores
+
+Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+
+- Chrome Web Store (for Google Chrome, Opera, Brave, Vivaldi, Microsoft Edge (>79), and other Chromium-based browsers)  
+  https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco  
+- Add-ons for Firefox (for Mozilla Firefox)  
+  https://addons.mozilla.org/firefox/addon/scratch-messaging-extension/  
+
+## From GitHub
+
+[The releases tab](https://github.com/ScratchAddons/ScratchAddons/releases) contains all builds of Scratch Addons, as well as the mirror of the store builds hosted on both servers.
+
+## From source
+
+### Cloning the repository
+
+This is the recommended way to install for development purposes. This assumes you have Git installed.
+
+To download the repository, simply clone `https://github.com/ScratchAddons/ScratchAddons.git`. Here's an example.
+
+```sh
+$ git clone https://github.com/ScratchAddons/ScratchAddons.git
+```
+
+If you have GitHub, you can fork the repository in GitHub beforehand, and then clone the fork instead.
+
+```sh
+$ git clone https://github.com/[your-name]/ScratchAddons.git
+```
+
+### Downloading the zipball
+
+If you don't have Git installed, you can try this method instead.
+
+1. Go to the [repository](https://github.com/ScratchAddons/ScratchAddons) and find the download code button
+
+![Screenshot 2020-09-04 at 6 41 12 PM](https://user-images.githubusercontent.com/61319150/92291572-f17ef080-eede-11ea-85e8-fda961a56074.png)
+
+2. Click it and select "Download ZIP"
+
+![Screenshot 2020-09-04 at 6 43 57 PM](https://user-images.githubusercontent.com/61319150/92291570-f0e65a00-eede-11ea-80ee-f32e340327d9.png)
+
+3. Extract the archive into a folder.
+
+### Installing on Google Chrome
+
+1. Go to the [Extension Management page](chrome://extensions) (`chrome://extensions`).
```

Comment:
The page is not web-navigatable, and I don't think GFM allows that protocol. Unlinkify.